apps/search/src/search/models.py

```diff
@@ -229,11 +229,14 @@ def create2(self, name, label, is_core_only=False, owner=None):
 
 class Collection(models.Model):
   """All the data is now saved into the properties field"""
+  ICON = 'search/art/icon_search_48.png'
+  
   enabled = models.BooleanField(default=False) # Aka shared
   name = models.CharField(max_length=40, verbose_name=_t('Solr index name pointing to'))
   label = models.CharField(max_length=100, verbose_name=_t('Friendlier name in UI'))
   is_core_only = models.BooleanField(default=False)
   cores = models.TextField(default=json.dumps({}), verbose_name=_t('Collection with cores data'), help_text=_t('Solr json')) # Unused
+  icon = models.URLField(default=ICON)
```

Comment:
When a new field is added, we would it added to the South Schema migrations with the ./build/env/bin/hue schemamigration command